js/components/AppPageComponent.jsx

```diff
@@ -109,8 +109,8 @@ var AppPageComponent = React.createClass({
     return (
       <div className="header-btn">
         <button className="btn btn-sm btn-default"
-            onClick={this.props.suspendApp}
-            disabled={this.props.model.get("instances") < 1}>
+          onClick={this.props.suspendApp}
+          disabled={this.props.model.get("instances") < 1}>
```

Comment:
We use the extra indentation to make it easy to read what belongs to the element definition, and what belongs inside. The normal indentation is only used when you close the element right away. Like so:
```html
<Element className="lol-burger"
    onClick={this.handleClick}
    disabled={this.isDisabled}>
  <NestedElement />
</Element>
```

```html
<Element className="lol-burger"
  onClick={this.handleClick}
  disabled={this.isDisabled} />
<SiblingElement />
```